### PR TITLE
Fix renovate config warnings

### PR DESCRIPTION
**Pull request by BacLuc**
See the single commits for more info

### CODE REVIEW

File: renovate.json [hunk 1]
```diff
@@ -1,6 +1,7 @@
 {
   "extends": [
-    "config:base",
+    "mergeConfidence:all-badges",
```
Comment (usu) [inline renovate.json:3]: Why is this needed? 
I understood, this would only need to be enabled for self-hosting renovate: https://docs.renovatebot.com/merge-confidence/#enabling-and-disabling

**Review by usu — COMMENTED** [review]
In the renovate logs, I can see another config warning:

image

Should we try to fix this as well in this PR?

File: renovate.json (hunk 1 above)
Comment (BacLuc) [reply renovate.json:3]: The massaged config contains the values that are in fact used.
One way is to make the config transparent. e.g. when i run renovate locally to test renovate, then this preset is not set.
But in both cases i would want the same output.
But it is not important, i can also drop this commit

**Comment by BacLuc** [comment]
@carlobeltrame 
@usu 
@pmattmann 
ping
there are other changes in the pipeline

**Review by carlobeltrame — APPROVED** [review]

**Review by usu — APPROVED** [review]
